### PR TITLE
Add TFLite accuracy estimation tool

### LINKED ISSUE #1852: TFLite accuracy tooling
Let's get a `evaluate_tflite.py` that allows to run something similar to `evaluate.py` but for  TFLite model.

### PR DESCRIPTION
**Pull request by lissyx**
Fixes #1852

### CODE REVIEW

**Comment by lissyx** [comment]
@reuben Should be better now

File: evaluate_tflite.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+from six.moves import zip, range
+from multiprocessing import JoinableQueue, Pool, Process, Queue, cpu_count
+from deepspeech import Model
+
+from util.text import levenshtein
+from evaluate_tools import process_decode_result, calculate_report
+
+r'''
+This module should e self-contained:
```
Comment (reuben) [inline evaluate_tflite.py]: typo: should *be*

File: evaluate_tflite.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  - build a libdeepspeech.so with TFLite:
```
Comment (reuben) [inline evaluate_tflite.py]: nit: build libdeepspeech.so

File: evaluate_tflite.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    - add a dep in native_client/BUILD against TFlite: '//tensorflow:linux_x86_64': [ "//tensorflow/contrib/lite/kernels:builtin_ops" ]
+    - bazel build [...] --copt=-DUSE_TFLITE [...] //native_client:libdeepspeech.so
```
Comment (reuben) [inline evaluate_tflite.py]: In the future, maybe we should make it a separate target, like `libdeepspeech_tflite.so`, to streamline this process.

File: evaluate_tools.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
```
Comment (reuben) [inline evaluate_tools.py]: We should put this in `util` along with the other modules.

File: evaluate_tflite.py (hunk 3 above)
Comment (lissyx) [reply evaluate_tflite.py]: Yeah, we could look into that.

**Pull request merged**